<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListFilterDialog</class>
 <widget class="QDialog" name="ListFilterDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>522</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Select items from a filterable list ...</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="2">
      <widget class="QListWidget" name="list_out">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>double-click an item to move it to the left</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string/>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <widget class="QPushButton" name="btn_left_right">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>160</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>30</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip">
         <string>move all selected items from left to right</string>
        </property>
        <property name="text">
         <string>&gt;&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_right_left">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>260</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>move all selected items from right to left</string>
        </property>
        <property name="text">
         <string>&lt;&lt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_left_right_all">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>200</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>move all items from left to right</string>
        </property>
        <property name="text">
         <string>&gt; ALL &gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_right_left_all">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>300</y>
          <width>61</width>
          <height>23</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>move all items from right to left</string>
        </property>
        <property name="text">
         <string>&lt; ALL &lt;</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="FilterableList" name="list_in" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>400</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>double-click an item to move it to the right</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Available</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Chosen</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok_button_">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel_button_">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FilterableList</class>
   <extends>QWidget</extends>
   <header>OpenMS/VISUAL/MISC/FilterableList.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
